csp2-screen.xml

```diff
@@ -1,46 +1,45 @@
-<tool id="csp2-screen" name="CSP2 (Screening Mode)" version="0.9.7_Dev21">
+<tool id="csp2-screen" name="CSP2 (Screening Mode)" version="0.9.7_Dev21" profile="23.0">
         <description>Screen query assemblies against reference assemblies</description>
         <requirements>
                 <requirement type="package" version="24.10.1">nextflow</requirement>
                 <requirement type="package" version="1.5.8">micromamba</requirement>
         </requirements>
         <version_command>nextflow -version</version_command>
         <command detect_errors="aggressive"><![CDATA[
-export CSP2_DIR=\$PWD;
-mkdir -p \$CSP2_DIR/queries \$CSP2_DIR/references;
+mkdir -p ./queries ./references &&
 
 if [ -n "$query_fasta" ] && [ "$query_fasta" != "None" ]; then
```

Comment:
This is bash, and will be executed during the runtime of the job, on your cluster.

I don't think this is what you want here?